src/index.ts

```diff
@@ -516,9 +518,7 @@ export const newForm: NewFormFn = <T extends object>(
           try {
             __isValidating.set(true);
 
-            await config.validationSchema.validate(currentValues, {
-              abortEarly: false,
-            });
+            await config.validationSchema.validate(currentValues, config.validationOption);
```

Comment:
For backwards compatibility it would be nice to fallback to current value.

Can you please provide current value if none is passed?

```suggestion
            await config.validationSchema.validate(currentValues, config.validationOptions ? config.validationOptions : { abortEarly: false });
```